pkg/controller/dnszone/dnszone_controller.go

```diff
@@ -50,6 +52,9 @@ var (
 	},
 		[]string{"force"},
 	)
+
+	// accessDeniedMessageRegexp is a regex used to fetch condition message from error when credentials are missing permission to create DNS zone
+	accessDeniedMessageRegexp = regexp.MustCompile(`^AccessDeniedException:\s+([^\t]+)\n\t`)
```

Comment:
What do we gain from restricting the substring to non-tabs? I would expect one of the two following.
```suggestion
	accessDeniedMessageRegexp = regexp.MustCompile(`^AccessDeniedException:\s+(.+)\n\t`)
```
```suggestion
	accessDeniedMessageRegexp = regexp.MustCompile(`^AccessDeniedException:\s+([^\t]+)`)
```